### PR TITLE
Prep work before fully moving queries to controller (can't find controller :c)

### PR DESCRIPTION
**Pull request by danielarlt**
### Please check if the PR fulfills these requirements:

* [ ] The PR title and message follows our [guidelines](https://submitty.org/developer/how_to_contribute)
* [ ] Tests for the changes have been added/updated (if possible)
* [ ] Documentation has been updated/added if relevant

### Other information?

This project was worked on by Daniel Arlt, Mason Wilde, and Andrew Binder.

We can't find the associated controller to move the queries to.

### CODE REVIEW

**Comment by nick97eagles** [comment]
Maybe I'm blind too, but I also couldn't find a controller linked specifically to this view. You might try checking the student or admin controller folders, it could be hidden in there.

**Comment by CalebJ2** [comment]
I think Submitty\site\app\controllers\student\SubmissionController.php and Submitty\site\app\controllers\NavigationController.php call this view's showGradeable function

File: site/app/views/submission/HomeworkView.php [hunk 1]
```diff
@@ -238,9 +238,11 @@ private function renderSubmitNotAllowedBox() {
      * @param GradedGradeable|null $graded_gradeable
      * @param AutoGradedVersion|null $version_instance
      * @param int $late_days_use
+     * @param $students
+     * @param $ggs
      * @return string
      */
-    private function renderSubmitBox(Gradeable $gradeable, $graded_gradeable, $version_instance, int $late_days_use): string {
+    private function renderSubmitBox(Gradeable $gradeable, $graded_gradeable, $version_instance, int $late_days_use, $students, $ggs): string {
```
Comment (dakriy) [inline site/app/views/submission/HomeworkView.php]: Any reason you don't want to specify the type here?

**Comment by andrewdbinder** [comment]
Out of scope, see #36

**Pull request closed without merging**